agent/scripts/newrelic.ini.template

```diff
@@ -1305,6 +1304,29 @@ newrelic.daemon.logfile = "/var/log/newrelic/newrelic-daemon.log"
 ;newrelic.application_logging.forwarding.context_data.include = ""
 ;newrelic.application_logging.forwarding.context_data.exclude = ""
 
+; Setting: newrelic.application_logging.forwarding.labels.enabled
+; Type   : boolean
+; Scope  : per-directory
+; Default: false
+; Info   : Toggles whether the agent adds labels as attributes to log records for 
+;          sending to New Relic.  The labels can be defined by the newrelic.labels
+;          configuration value, and filtered using the 
+;          newrelic.application_logging.forwawrding.labels.exclude configuration value.
+;
+;newrelic.application_logging.forwarding.labels.enabled = false
+
+; Setting: newrelic.application_logging.forwarding.labels.exclude
+; Type   : string
+; Scope  : per-directory
+; Default: ""
+; Info   : A list of labels to NOT add as attributes to logs which are forwarded
+;          to New Relic.  This list can be separated by commas.
```

Comment:
Please update description to state that label matching is case insensitive and multiple list elements *MUST* be separated by commas.

Reply:
An example (like [regular labels provides](https://github.com/newrelic/newrelic-php-agent/blob/main/agent/scripts/newrelic.ini.template#L863C18-L863C50)) could also be useful.

Reply:
Updated in commit https://github.com/newrelic/newrelic-php-agent/pull/1027/commits/277d03449135311cb9dd9d0661d8eaa0982f69e4.